hugo/assets/scss/components/columns.scss

```diff
@@ -31,6 +37,28 @@
                     height: 100%;
                 }
             }
+
+            &:first-child:nth-last-child(1) {
+                max-width: 100%;
+            }
+
+            &:first-child:nth-last-child(2),
+            &:first-child:nth-last-child(2) ~ #{ $self }__col {
+                --number-of-cols: 2;
```

Comment:
Looks awesome! But I think you also need to add `flex: 0 0 50%` in this case and 33.33% and 25% in the cases below. This way the code blocks fill the width of the page, even when there is less content